### PR TITLE
feat: --no-mount for 'bind path' entries, from sylabs 626

### PR DESCRIPTION
**Pull request by edytuk**
This pulls in sylabs PR

- sylabs/singularity#626
which fixed
- sylabs/singularity#625

The original PR description was:
> Allow one or more `bind path` entries in `singularity.conf` to be
> ignored by specifying the absolute path of the destination as a
> `--no-mount` or `SINGULARITY_NO_MOUNT` value.
> 
> This allows finer-grained control of bind mounts than `--contain`,
> which will disable all.
> 
> It is useful in situations where e.g. a directory `/project` in a
> container clashes with a `bind path = /project` entry, but other
> default binds from the host are still required.
> 
> Note that `/etc/hosts` and `/etc/localtime` were always specially
> injected with `--contain`, but the new `--no-mount` handling now
> allows this to be overridden also.

### CODE REVIEW

**Comment by DrDaveD** [comment]
Closing & reopening to kick off CI.

File: pkg/runtime/engine/apptainer/config/config.go [hunk 1]
```diff
@@ -498,6 +499,16 @@ func (e *EngineConfig) GetNoCwd() bool {
 	return e.JSON.NoCwd
 }
 
+// SetSkipBinds sets bind paths to skip
+func (e *EngineConfig) SetSkipBinds(val []string) {
+	e.JSON.SkipBinds = val
+}
+
+// GetSkipBinds gets bind paths to skip
+func (e *EngineConfig) GetNoBinds() []string {
```
Comment (cclerget) [inline pkg/runtime/engine/apptainer/config/config.go:508]: Please rename to `GetSkipBinds` for consistency as it is also set in function comment

**Review by cclerget — CHANGES_REQUESTED** [review]

File: pkg/runtime/engine/apptainer/config/config.go (hunk 1 above)
Comment (edytuk) [reply pkg/runtime/engine/apptainer/config/config.go]: I renamed function and parameter for consistency. Is it ok?

Comment (cclerget) [reply pkg/runtime/engine/apptainer/config/config.go]: :+1: thanks @edytuk !

**Review by cclerget — APPROVED** [review]

**Pull request merged**